kodilla-spring/src/main/java/com/kodilla/spring/portfolio/TaskList.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.spring.portfolio;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class TaskList
+{
+    private List<String> tasks;
+
+    public TaskList()
+    {
+        this.tasks = new ArrayList<>();
```

Comment:
= new ArrayList<>(); 
to może polecieć do linii 8 i wtedy konstruktor jest zbędny.

Reply:
Wiem, ale polecenie brzmiało:
"Jej zawartość powinna być inicjowana w konstruktorze przy pomocy polecenia tasks = new ArrayList<>();". Dlatego zostawiłem konstruktor.